content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade.md

```diff
@@ -378,6 +378,20 @@ This command is idempotent and eventually makes sure that the actual state is th
 
 To recover from a bad state, you can also run `kubeadm upgrade apply --force` without changing the version that your cluster is running.
 
+During upgrade kubeadm writes the following backup folders under `/etc/kubernetes/tmp`:
+- `kubeadm-backup-etcd-<date>-<time>`
+- `kubeadm-backup-manifests-<date>-<time>`
+
+`kubeadm-backup-etcd` contains a backup of the local etcd member data for this control-plane Node.
+In case of an etcd upgrade failure and if the automatic rollback does not work, the contents of this folder
+can be manually restored in `/var/lib/etcd`. In case of HA setup with multiple local etcd members
```

Comment:
Are we skipping the etcd backed in case of multi-controlplane... I don't remember this detail 😳

Reply:
before writing this update, digging the source code i found that we clean the etcd local member backup in case of external etcd or HA:
https://github.com/kubernetes/kubernetes/blob/ce2102f3637134519ec189f096da5277af6072a6/cmd/kubeadm/app/phases/upgrade/staticpods.go#L595

https://github.com/kubernetes/kubernetes/blob/master/cmd/kubeadm/app/phases/upgrade/staticpods.go#L102

https://github.com/kubernetes/kubernetes/blob/ce2102f3637134519ec189f096da5277af6072a6/cmd/kubeadm/app/phases/upgrade/staticpods.go#L182

> In case of HA setup with multiple local etcd members

perhaps i need to clarify that the backup is always written, but after upgrade:
- the folder contents will be cleaned for HA setup.
- the folder contents will persist for single control-plane setup.

or maybe we need to change this logic and always keep the backup.

Reply:
I think that making consistent how we managed stacked etcd (HA or not) makes more sense

Reply:
this has to be a k/k PR.
in my opinion we should always persist the etcd backup for stacked.

i'm inclined to remove the last sentence from this k/website PR:
> In case of HA setup with multiple local etcd members ... 

WDYT?

Reply:
ok to remove the last sentence and open an issue (a PR) to track the change on update

Reply:
@fabriziopandini 
i've updated the PR to only have the "external etcd" part of the sentence.
created https://github.com/kubernetes/kubeadm/issues/1998 to track the k/k cleanup/

Reply:
/hold
until the discussion in https://github.com/kubernetes/kubeadm/issues/1998 is resolved.